conversion/conversion.go

```diff
@@ -417,6 +417,70 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+func VerifyTargetDb(targetDb, project, instance, dbName string) (string, bool, error) {
+	ctx := context.Background()
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return "", false, fmt.Errorf("can't create admin client: %w", analyzeError(err, project, instance))
+	}
+	defer adminClient.Close()
```

Comment:
you should do this before "if err != nil" else, the function will return before issuing the defer and thus before closing the adminClient

Reply:
But in case an error occurs, won't the adminClient not get created in the first place so no need to close it?